src/backend/fs.rs

```diff
@@ -37,26 +47,22 @@ pub async fn write_file(pk: Secp256k1PubKey, file_bytes: &[u8]) -> Result<Blake3
     encoded_segments.push(last_segment);
 
     trace!("Get eight storage volumes from config");
-    if SYS_CFG.volumes.len() != 8 {
+    if SYS_CFG.volumes.len() != FEC_M {
         return Err(anyhow!("Eight volume paths must be configured"));
     }
 
-    trace!("Create a shared secret using ECDH");
-    let sk = SYS_CFG.private_key;
-    let ss = SharedSecret::new(&pk.into_inner(), &sk);
-
     trace!("Split each segment out into 8 separate chunks and write each chunk to the storage volume by filename");
     let segment_hashes = encoded_segments
         .par_iter()
         .map(|encoded_segment| {
             let Encoded(encoded_bytes, bao_hash, encode_info) = encoded_segment;
             trace!("Encoded bytes len: {}", encoded_bytes.len());
 
-            let encoded_chunk_size = encode_info.bytes_verifiable / 8;
+            let encoded_chunk_size = encode_info.bytes_verifiable as usize / FEC_M;
```

Comment:
What is the purpose of "as usize" in this case...Is it required?

Reply:
Yes, because FEC_M is defined as a usize, but bytes_verifiable is defined as a u32. See:
https://docs.rs/carbonado/latest/carbonado/constants/constant.FEC_M.html
and
https://docs.rs/carbonado/latest/carbonado/structs/struct.EncodeInfo.html